src/routes/Board.svelte

```diff
@@ -10,6 +10,19 @@
 	];
 
 	initPieces($boardArr);
+
+	const handleMove = (squareNumber: number, occupier: string) => {
+		if (squareNumber !== $selectedSquare && occupier != '') {
```

Comment:
Maybe define this as as variables, so it easier to understand the logic?

Reply:
done